app/feature/travel-certificate/src/main/res/xml/file_paths.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<paths>
```

Comment:
Same comment as the one above, we currently have this https://github.com/HedvigInsurance/android/blob/41758c154b9483826864a5f67f0f301caa053054/app/app/src/main/res/xml/provider_paths.xml#L25-L27. Again, not sure if we can reuse that (even if it means adding this new path as an entry there) so we don't have multiple ways to do this and confuse ourselves.

Reply:
Cool, Ill try to see if we can use this instead

Reply:
Didnt quite get this to work since we needed to then add a dependecy on the app module from the travel cert module... I think its fine to have these separated by feature though. Even though it makes for a bit more and duplicated code.

Reply:
Oh yeah absolutely. Optimally it'd be that the chat feature was also in a separate module, and then this could be shared somehow or something like that. But yeah let's go ahead like this now, totally forgot about that here.